src/testers/unittests/test_semantics.py

```diff
@@ -149,6 +149,27 @@ def emulate(self, pc):
 
             self.assertTrue(ret)
 
+            try:
```

Comment:
Here you only check that str() call doesn't crash/raise exception?
May be you can add in a test_utils.py this as a : check_instruction_data_is_valid() or something like this?